dev/bots/prepare_package/archive_publisher.dart

```diff
@@ -49,15 +49,15 @@ class ArchivePublisher {
   static String getMetadataFilename(Platform platform) => 'releases_${platform.operatingSystem.toLowerCase()}.json';
 
   Future<String> _getChecksum(File archiveFile) async {
-    final DigestSink digestSink = DigestSink();
+    final AccumulatorSink<Digest> digestSink = AccumulatorSink<Digest>();
     final ByteConversionSink sink = sha256.startChunkedConversion(digestSink);
 
     final Stream<List<int>> stream = archiveFile.openRead();
     await stream.forEach((List<int> chunk) {
       sink.add(chunk);
     });
     sink.close();
-    return digestSink.value.toString();
+    return digestSink.events.single.toString();
```

Comment:
Is it safe to call `events.single` here? From reading the code, it looks like we're adding multiple chunks to it.

Reply:
My understanding was that all those chunks get turned into a single `Digest` though.

Reply:
I'm requesting changes while I play with this to confirm--will get back to you after testing...

Reply:
FWIW, this is also what is suggested in the readme of the crypto package: https://pub.dev/packages/crypto#digest-on-chunked-input

Reply:
yep you're right. I missed that `sink` is not `digestSink`, LGTM